phpspec.yml.dist

```diff
@@ -0,0 +1,4 @@
+suites:
+    main:
+        namespace: SitemapPlugin
+        psr4_prefix: SitemapPlugin
```

Comment:
Missing newline

Reply:
Downside of not having `.editorconfig` in the repos :|